### PR TITLE
Pep8 on script

### PR DESCRIPTION
**Pull request by wasade**
Wanted to get a bit more involved in EMPeror, and needed an activity that wasn't too mentally taxing.

### CODE REVIEW

**Comment by emperor-helper** [comment]
The test build for this pull request can be found here: http://emperor.colorado.edu/pull_372/make_emperor/

File: scripts/make_emperor.py [hunk 1]
```diff
@@ -277,62 +378,71 @@ def main():
     serial_comparison = True
 
     # can't do averaged pcoa plots _and_ custom axes in the same plot
-    if custom_axes!=None and len(custom_axes.split(','))>1 and\
-        isdir(input_coords):
-        option_parser.error(('Jackknifed plots are limited to one custom axis, '
-            'currently trying to use: %s. Make sure you use only one.' %
-            custom_axes))
+    if custom_axes is not None and isdir(input_coords):
+        if custom_axes.count(',') > 0:
```
Comment (ElDeveloper) [inline scripts/make_emperor.py]: Fancy!

File: scripts/make_emperor.py [hunk 2]
```diff
@@ -2,236 +2,337 @@
...
-    format_emperor_autograph)
+                            format_taxa_to_js, format_vectors_to_js,
+                            format_emperor_html_footer_string,
+                            format_comparison_bars_to_js,
+                            EMPEROR_HEADER_HTML_STRING, EmperorLogicError,
+                            format_emperor_autograph)
 from emperor.parse import parse_coords
 
+
+__author__ = "Antonio Gonzalez Pena"
```
Comment (ElDeveloper) [inline scripts/make_emperor.py]: Can you remove all of these, except for the __version__ string?

**Comment by ElDeveloper** [comment]
Looks good, just one minor thing.

File: scripts/make_emperor.py (hunk 2 above)
Comment (wasade) [inline scripts/make_emperor.py]: done

File: scripts/make_emperor.py (hunk 1 above)
Comment (wasade) [inline scripts/make_emperor.py]: :bow:

**Comment by wasade** [comment]
Should be good to go

**Comment by ElDeveloper** [comment]
Thanks both!

**Pull request merged**